gensim/topic_coherence/__init__.py

```diff
@@ -1,4 +1,3 @@
 """
-This package contains implementation of the individual components of
-the topic coherence pipeline.
+Individual components of the topic coherence pipeline.
```

Comment:
question about `__init__.py`: this type of files will be suggested in reference? If yes - how this will looks like?

Reply:
It will be printed out in the apiref_nextgen.rst